### PR TITLE
Navigation and styling

### PR DESCRIPTION
**Pull request by janabremer**
I added a navigation, a landing page and styling to my app.
It looks more than it is - please ignore the svgs and the snapshots!

As always looking forward to your feedback! 🙂

### CODE REVIEW

File: frontend/src/pages/ProductPage.js [hunk 1]
```diff
@@ -25,24 +28,21 @@ export default function ProductPage({productId, lastProduct, nextProductPage, on
...
+            <ProductInfo name={product.name} water={product.water} apiState={productState} />
+            {lastProduct && nextProductPage && <Button text={'next'} onClick={onLoadNext} iconComponentRight={<ArrowRightIcon />} />}
+            {firstProduct && prevProductPage && <Button text={'previous'} onClick={onLoadPrev} iconComponentLeft={<ArrowLeftIcon />} />}
+            {photoState === 'ERROR' ? <PhotoSource /> : <PhotoSource photographer={photo.photographer} plattform={photo.plattform} url={photo.pexelsUrl} />}
+        </ProductPageStyled>
     )
 }
 
-const PageStyled = styled.main`
+const ProductPageStyled = styled(PageStyled)`
```
Comment (marwinburesch) [inline frontend/src/pages/ProductPage.js]: `ProductPageStyled` is a styled `PageStyled` 😁 
and `PageStyled` is already in the folder styles, maybe you can omit some of these "style"

File: frontend/src/styles/PageStyled.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components/macro'
+
+const PageStyled = styled.main`
```
Comment (marwinburesch) [inline frontend/src/styles/PageStyled.js]: you could just name this `Page` , `BasePage` or `PageContainer`

File: frontend/src/components/SearchBar.js [hunk 3]
```diff
@@ -34,29 +39,29 @@ export default function SearchBar({onSearch}) {
...
 
-const InputField = styled.input`
+const InputFieldStyled = styled.input`
     background-color: var(--primary-light-transparent);
-    border-radius: 25px;
     border: none;
-    /* border: solid 2px var(--secondary-dark); */
-    box-shadow:  0px 0px 20px var(--shadow-dark);
+    box-shadow:  0 0 20px var(--shadow-dark);
+    caret-color: var(--primary-blue);
```
Comment (marwinburesch) [inline frontend/src/components/SearchBar.js:50]: uuuuuh didn't know that one, cool! :)

File: frontend/src/components/PhotoSource.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+PhotoSource.propTypes = {
+    protographer: PropTypes.string.isRequired,
+    plattform: PropTypes.string,
+    link: PropTypes.string
```
Comment (marwinburesch) [inline frontend/src/components/PhotoSource.js]: This is supposed to be `url` I think

File: frontend/src/components/PhotoSource.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    <PhotoSourceStyled href={url} target="_blank">Photo by {photographer} on {plattform}</PhotoSourceStyled>
+    )
+}
+
+const PhotoSourceStyled = styled.a`
+    color: var(--primary-light-transparent);
+    font-size: 50%;
+    padding: 0 10px 10px 0;
+    place-self: end;
+    text-decoration: none;
```
Comment (marwinburesch) [inline frontend/src/components/PhotoSource.js]: I think this could use some indication that it is a link actually

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Excellent job, looks very clean! 🐜 💯 
Maybe work on the non-mobile view now? You could just a quick container that sets constraints for the App, just so it looks less awkward when you open the site without dev tools.
(RC because of the PropTypes)

Cheers!

File: frontend/src/pages/ProductPage.js (hunk 1 above)
Comment (janabremer) [reply frontend/src/pages/ProductPage.js]: renaming all my pageStyled and componentStyled will be part of my next refactoring user story 😅

File: frontend/src/components/PhotoSource.js (hunk 4 above)
Comment (janabremer) [reply frontend/src/components/PhotoSource.js]: yes, thanks!

**Pull request merged**